pybricksdev/ble.py

```diff
@@ -102,7 +103,7 @@ async def connect(self, device: BLEDevice):
         """
 
         print("Connecting to", device)
-        self.client = BleakClient(device)
+        self.client = BleakClient(device.address)
```

Comment:
By passing the address instead of device object will cause Bleak to have to scan for the device again. This would make connections much slower.